openpaas-james/jmap/extensions/src/main/scala/com/linagora/openpaas/james/jmap/method/KeystoreSetCreatePerformer.scala

```diff
@@ -64,7 +65,7 @@ class KeystoreSetCreatePerformer @Inject()(serializer: KeystoreSerializer,
   private def createKey(mailboxSession: MailboxSession,
                         clientId: KeystoreCreationId,
                         keystoreCreationRequest: KeystoreCreationRequest): SMono[KeystoreCreationResult] =
-      SMono.fromPublisher(keystore.save(mailboxSession.getUser, keystoreCreationRequest.key))
+      SMono.fromPublisher(keystore.save(mailboxSession.getUser, Base64.decode(keystoreCreationRequest.keyBase64)))
```

Comment:
That was done too fast, I should refactor this base64 decode part I think

Reply:
You should rather get rid of this base64 stuff indeed.

Reply:
Same as above then, if you manage to pass the key as text in your json let me know how